zero_to_chat_with_your_data/zero_to_chat_with_your_data.md

```diff
@@ -117,9 +117,9 @@ Under **Projects** on the left-hand panel, select the **Notebooks** tab.
 ### Notebooks Structure
 
 >  **Notebooks vs. the UI**
-Most of the exercises in this lab are executed using pre-written SQL within the notebook to save time. These tasks can also be done via the UI, but would require navigating back-and-forth between multiple UI tabs.
+To save time, the tasks in this notebook are executed using pre-written SQL. These tasks could also be completed through the UI, but would require navigating between multiple tabs
```

Comment:
Sorry, that was the original direction but we are using worksheets instead. Can you please adjust the text accordingly?

Reply:
I added this note in the worksheet section. Let me know if it's sufficient.
![image](https://github.com/user-attachments/assets/27f9cfaf-dd9c-4d32-bf72-19565af29569)